src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      return this.result + number;
+    },
```

Comment:
```suggestion
    add(number) {
      return this.result + number;
    },
```
you can get rid of all these returns, and adjust result's value right here